spec/factories/users.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :user do
+    user_id { '000000'}
```

Comment:
should this be a `SecureRandom.uuid`? Just wondering in case there will be clashes in the tests . . .

Reply:
doesn't need to... `user` is not represented in the database in any way and there should only be one instance of `user` at any time...  (it's same like below with name and email)